hrabsence/hrabsence.php

```diff
@@ -587,3 +589,16 @@ function hrabsence_civicrm_pageRun($page) {
       ->addStyleFile('org.civicrm.hrabsence', 'css/hrabsence.css');
   }
 }
+
+/**
+ * Remove any defined relationship type by this extension
+ *
+ */
+function _hrabsence_removeRelationshipTypes() {
```

Comment:
Wouldn't it be better to use managed entities for this? Is it even possible possible?